Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -702,18 +707,54 @@ Don't have a 4chan Pass?<br>
         <item>Only posts quoting you</item>
     </string-array>
 
-    <string-array name="setting_watch_sounds">
-        <item>All posts</item>
-        <item>Only posts quoting you</item>
-    </string-array>
-    <string-array name="setting_watch_leds">
-        <item>None</item>
-        <item>White</item>
-        <item>Red</item>
-        <item>Yellow</item>
-        <item>Green</item>
-        <item>Cyan</item>
-        <item>Blue</item>
-        <item>Purple</item>
-    </string-array>
+    <string name="import_or_export_dialog_title">Overwrite existing file or create a new one?</string>
+    <string name="import_or_export_dialog_positive_button_text">Overwrite</string>
+    <string name="import_or_export_dialog_negative_button_text">Create new</string>
+    <string name="import_or_export_warning">Warning!</string>
+    <string name="import_or_export_warning_super_long_message">You have %1$s %2$s %3$s set to be located in a directory that uses Storage Access Framework. Such directory locations cannot be exported into the settings file because SAF is fucking retarded (Basically if you try to export them and then use on another phone or if you delete the app then install it and import those settings the directory locations they are pointing to WILL NOT BE VALID ANYMORE AND THE APP WILL CRASH). Thus THEY WILL BE RESET TO DEFAULT in the exported settings file! So once you import them you will have to manually set back those locations in the Media Settings (so that the app can work with them). Sorry for such an inconvenience but there is nothing we can do. \n\nBlame Google for forcing this retarded fucking shit.</string>
```

Comment:
Get rid of the profanity dude. Also this is way too long of a string for a single line, spread it out over multiple lines in the file.